docs-web/installation/building-ingress-controller-image.md

```diff
@@ -22,17 +22,17 @@ We build the image using the make utility and the provided `Makefile`. Let’s c
 1. Clone the Ingress controller repo:
     ```
     $ git clone https://github.com/nginxinc/kubernetes-ingress/
-    $ git checkout v1.6.1
+    $ cd kubernetes-ingress
+    $ git checkout v1.6.2
     ```
 
 1. Build the image:
     * For NGINX:
       ```
-      $ make clean
```

Comment:
no need to delete make clean. Changes to the Makefile have been made to the edge version, not release version.
same in the commands for Plus